static/js/components/search/CountryRefinementOption.js

```diff
@@ -10,17 +10,22 @@ export default class CountryRefinementOption extends React.Component {
     count:    number,
   };
 
-  countryName = (countryCode: string): string => (
-    iso3166.country(countryCode).name
-  );
+  countryName = (countryCode: string): string => {
+    let country, countryName = "-";
+    if (countryCode) {
+      country = iso3166.country(countryCode);
+      countryName = country ? country.name : "-";
+    }
+    return countryName;
```

Comment:
Ah, it will do that if the country code is `null` or `undefined`. if you call `iso3166.country('')` it just returns an empty object though, so maybe instead we could do:

```js
let country = iso3166.country(R.defaultTo("", countryCode)); 
return R.pathOr("-", ['name'], country);
```

alternatively, there's a little helper function in `lib/currency.js` that should be rewritten! It would also have this bug. We could rewrite it as:

```js
const codeToCountryName = R.compose(
  R.pathOr("-", ['name']), iso3166.country, R.defaultTo("")
);
```

You might need to make slight changes to that, but I think it should work. Then you could just import that function here and use it (and maybe also in the other file you changed for this PR).